meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,243 @@
+                    "matchType": "noExactMatch",
+                    "score": 0.3333333333333333
+                }
+            }
+```
+
+This experimental feature can be turned on at runtime using v1.3.0's new `/experimental-features` endpoint.
+```bash
+ curl \
+  -X PATCH 'http://localhost:7700/experimental-features/' \
+  -H 'Content-Type: application/json'  \
+--data-binary '{
+    "scoreDetails": true
+  }'
+
+```
+
+🗣️ Since this feature is experimental, we need your help to iterate on it! You are more than welcome to share your thoughts and your feedback on [this discussion](https://github.com/meilisearch/product/discussions/674).
+
+⚠️ The experimental feature API can break between Meilisearch versions. Be careful if using it in production.
+
+Done by @dureuill in #3771
+
+## Define fields to search on at search time
+
+v1.3.0 introduces a new search parameter, `attributesToSearchOn`.
+
+`attributesToSearchOn` is a search parameter accepting an array of strings indicating one or more document attributes. Queries using `attributesToSearchOn` will restrict the search to the indicated attributes. Note the chosen attributes must be part of the [searchable attributes](https://www.meilisearch.com/docs/learn/configuration/displayed_searchable_attributes#the-searchableattributes-list) in settings.
+
+Given the following dataset:
+```json
+{
+  "id": 0,
+  "name": "Our Wives Under the Sea",
+  "genre": ["horror", "scifi"],
+  "synopsis": "A woman returns to her wife transformed after a deep-sea adventure."
+},
+{
+  "id": 1,
+  "name": "A Strange and Stubborn Endurance",
+  "genre": ["adventure"],
+  "synopsis": "A man must overcome trauma and fight off a mysterious assassin."
+}
+```
+And the following query:
+```json
+{
+  "q": "adventure",
+  "attributesToSearchOn": ["genre"]
+}
+```
+Meilisearch will only return document 1.
+
+Both documents contain the word adventure, but `"attributesToSearchOn": ["genre"]` instructs Meilisearch to only consider results found on the genre field.
+
+
+Done by @ManyTheFish in (#3834)
+
+
+## Search for facet values
+Meilisearch v1.3.0 introduces a new endpoint  `POST /indexes/{index}/facet-search` that allows you to search for facet values. Only fields defined as `filterableAttributes` will be facet-searchable. Facet search supports [prefix search](https://www.meilisearch.com/docs/learn/advanced/prefix) and [typo tolerance](https://www.meilisearch.com/docs/learn/configuration/typo_tolerance).
+
+Done by @Kerollmops in (#3699)
+
+## Sort facet values by count
+By default, all facet values are sorted in ascending alphanumeric order. With the release of v1.3.0 you can change this using the `sortFacetValuesBy` property of the [`faceting` index settings](https://www.meilisearch.com/docs/reference/api/settings#faceting). This will allow you to sort facet by value count i.e. the genre with the highest number of results shows first.
+
+Done by @Kerollmops in (#3612)
+
+### Performance improvements
```

Comment:
Can be removed and integrated into the "Other improvement" section

Reply:
We usually don't do "performance improements" section -> we did it for v1.2 because we had a lot to put into it thanks to the search refactor

Reply:
ah cool thanks! will update

Reply:
thats gone now and moved into other improvements

@Kerollmops tagging you just because one of your comments was to move something to the performance block. I've just pulled it all into Other improvements now as above.